index.html

```diff
@@ -43,36 +43,36 @@ <h1 class="binabox">
       </div>
       <section id="catalog">
         <h2 class="catalog" >Catalog</h2>
-        <button class="catalog-button">&lt;</button>
-        <button class="catalog-button">&gt;</button>
-        <button class="catalog-button__see">See all</button>
+        <button class="catalog__button">&lt;</button>
+        <button class="catalog__button">&gt;</button>
+        <button class="catalog__button--see">See all</button>
         <article class="product-card">
           <img class="product-card__image"src="skeleton.png" alt="Skeleton NFT" />
-          <span class="badge-nft"> new </span>
+          <span class="product-card__badge"> new </span>
           <p class="product-card__price">$199</p>
           <p class="product-card__name">Name Product</p>
         </article>
         <article class="product-card">
           <img class="product-card__image"src="Cat.png" alt="Cat NFT" />
-          <span class="bafge-nft">new</span>
+          <span class="product-card__badge">new</span>
           <p class="product-card__price">$199</p>
           <p class="product-card__name">Name Product</p>
         </article>
         <article class="product-card">
           <img class="product-card__image"src="scissors.png" alt="Scissors NFT" />
-          <span class="badge-nft__soldout">sold out</span>
+          <span class="product-card__badge--soldout">sold out</span>
```

Comment:
"product-card__badge product-card__badge--soldout" and the same for other modifiers